packages/foreman/foreman/foreman.spec

```diff
@@ -670,6 +670,7 @@ Group:  Applications/System
 # start specfile service Requires
 Requires: %{?scl_prefix}rubygem(puma) >= 5.1
 Requires: %{?scl_prefix}rubygem(puma) < 6.0
+Requires: %{?scl_prefix}rubygem(puma-status)
```

Comment:
I don't see this reflected in foreman's bundler.d file so on an update it will be removed. If it should be tracked just in packaging it should be outside of `start/end specfile service Requires`.

Reply:
Since this a more optional production helper, we had thought to not including in bundler and keep it outside. So I'll move it.